host/4/bullseye/amd64/java/java11-zulu/java11-core-tools.Dockerfile

```diff
@@ -0,0 +1,59 @@
+#-------------------------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See https://go.microsoft.com/fwlink/?linkid=2090316 for license information.
+#-------------------------------------------------------------------------------------------------------------
+
+FROM mcr.microsoft.com/java/maven:11-zulu-debian10
```

Comment:
Looks not bullseye and zulu. Is it expected?

Reply:
Zulu fine. See folder. Same thing though this is stuck on Buster it seems